src/interface/api_template/components/schemas/Participant.yaml

```diff
@@ -0,0 +1,8 @@
+title: Participant
```

Comment:
hmm. you shouldn't need to rewrite these schemas if they are apart of the fspiop/thirdparty and in snippets. is there some schema mismatch that I'm not seeing?

Reply:
Yeah this was leftover from Sridhar's original work updating the types and moving from what our interns had already written. I suspect I should be able to delete a lot of these components and update them to references to api-snippets.

I'll take care of that in a followup PR (see the todos in `mojaloop-template.yaml`)